src/main/java/com/aviva/CarRecommendations/InterestFilter.java

```diff
@@ -0,0 +1,65 @@
+package com.aviva.CarRecommendations;
+
+import com.aviva.DataAccess.BankingDataProcess;
+import com.aviva.Entities.AccountHolder;
+import com.aviva.Entities.Car;
+import org.json.JSONArray;
+import org.json.JSONObject;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+
+public class InterestFilter {
+
+    public JSONObject getBestFiveCars(String accountNumber) {
+        BankingDataProcess bdpInit = new BankingDataProcess();
+        AccountHolder user = bdpInit.getAccountHolder(accountNumber);
+
+        EstimateBudget ebInit = new EstimateBudget();
+        float yearlyBudget = ebInit.calculateYearlyBudget(user);
+        user.setSavings(yearlyBudget);
```

Comment:
fyi, savings is going to get removed when An merges her entity change, so maybe add a TODO to remind us to fix this later